src/components/App/App.js

```diff
@@ -15,15 +14,15 @@ function App() {
     const KEY = 'G3mzZaPrADRWteI33jNxbTRAM6TCT5NVzlc7za79';
 
     useEffect(() => {
-        console.log('running');
         const date = new Date();
-        const day = date.getDate();
-        const month = date.getMonth() + 1;
+        let day = date.getDate();
+        let month = date.getMonth() + 1;
```

Comment:
this could stay as const and change the way the pad is added:

```suggestion
        const day = date.getDate().toString().padStart(2, 0);
        const month = (date.getMonth() + 1).toString().padStart(2, 0);
```